src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -219,6 +219,12 @@ public function buildView(FormView $view, FormInterface $form, array $options)
             // POST request.
             $view->vars['full_name'] .= '[]';
         }
+
+        // Provide a default message when choices var is empty
+        if (empty($view->vars['choices']) && $options['expanded']) {
```

Comment:
:+1: for removing the expanded condition.

But I don't think it's worth init a view var `empty_choices` and add it to the context if `choices` are not empty.